engine/src/main/java/org/entando/entando/web/userprofile/ProfileTypeController.java

```diff
@@ -78,8 +80,10 @@ public void setProfileTypeValidator(ProfileTypeValidator profileTypeValidator) {
     @RestAccessControl(permission = Permission.SUPERUSER)
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<RestResponse> getUserProfileTypes(RestListRequest requestList) throws JsonProcessingException {
+        this.getProfileTypeValidator().validateRestListRequest(requestList);
         PagedMetadata<EntityTypeShortDto> result = this.getUserProfileTypeService().getShortUserProfileTypes(requestList);
         logger.debug("Main Response -> " + new ObjectMapper().writeValueAsString(result));
```

Comment:
Add toString method and parameterize so that the result only gets generated when debug is enabled.